src/Components/Generic/PopupComponents/PopUpFormTemplates/MainPopUpFormTemplate.js

```diff
@@ -0,0 +1,91 @@
+import Grid from '@material-ui/core/Grid';
+import React from 'react';
+import { connect } from 'react-redux';
+import { StyledRoundButton } from 'Components/Generic/PopupComponents/PopUpFormTemplates/Style';
+import { useTranslation } from 'react-i18next';
+import PopUpContantList from 'Components/Generic/PopupComponents/PopUpFormTemplates/PopUpContantList';
+import PopUpContext from 'Components/Generic/PopupComponents/PopUpFormTemplates/PopUpContext';
+import l from 'Assets/Images/l.png';
```

Comment:
please replace the image name to meaningful name :)

Reply:
I don't understand this is meanigful this is the main of the two components below ?

Reply:
what is 1.png ?

Reply:
I don't understand where do you see 1.png as we talked it is l.png not 1.png and it comes from here 
![image](https://user-images.githubusercontent.com/18376627/85232529-18fa0d00-b408-11ea-92ca-621b0f890756.png)
I can change it to left.png but please for further development talk to the designer to call the names by the names you like so we don't need to change it but only to take it from the zeplin.

Reply:
FIXED...